packages/react-common/src/moratorium-banner.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+
+export const CovidMoratoriumBanner: React.FC<{ locale?: string }> = ({
+  locale,
+}) => {
+  return locale === "es" ? (
+    <>
+      <b>Actualización COVID-19: </b>
+      JustFix.nyc está operativo, y hemos adaptado nuestros productos a las
+      normas establecidas durante la crisis de COVID-19. Todavía recomendamos
+      que se tomen todas las precauciones posibles para mantenerse sanos durante
+      esta crisis de salud pública. Gracias al poder de la organización de
+      inquilinos, los inquilinos no pueden ser desalojados por ninguna razón.
+      Visita las{" "}
+      <a
+        href="https://www.righttocounselnyc.org/ny_eviction_moratorium_faq"
```

Comment:
Do we actually want this to link to https://www.righttocounselnyc.org/moratoria_de_desalojo ?  That seems to be the spanish version, might save them having to find the link to the spanish version on the page...

Reply:
Oh yes good point!